app/src/main/java/org/simple/clinic/home/overdue/appointmentreminder/AppointmentReminderSheet.kt

```diff
@@ -110,6 +111,19 @@ class AppointmentReminderSheet : BottomSheetActivity() {
     decrementDateButton.isEnabled = state
   }
 
+  private fun textForAppointmentReminder(timeAmount: Int, chronoUnit: ChronoUnit): String {
+    val quantityStringResourceId = when (chronoUnit) {
+      ChronoUnit.DAYS -> {
```

Comment:
Forgot to leave this comment in the last review. These braces are unnecessary since the block is only one line. I think we can remove them. What do you think?